bayesian/api_v1.py

```diff
@@ -416,6 +417,47 @@ def post():
         return {'status': 'success'}
 
 
+class UserIntent(ResourceWithSchema):
+    method_decorators = [login_required]
+
+    @staticmethod
+    def get(user, ecosystem):
+        if user is not None and ecosystem is not None:
```

Comment:
This looks like a logical error. What do you want to accomplish with this check?

Reply:
i want the user to send me both user and ecosystem

Reply:
> i want the user to send me both user and ecosystem

OK, so in that case, I guess something like:

```python
if not user or not ecosystem:
```

should do the trick (empty strings). But I wonder if such can occur on flask level (not sure about it).